cardano-cli/src/Cardano/CLI/Types/Common.hs

```diff
@@ -645,3 +651,12 @@ data MustCheckHash a
   = CheckHash
   | TrustHash
   deriving (Eq, Show)
+
+data PotentiallyCheckedAnchor anchorType anchor
+  = PotentiallyCheckedAnchor
+  { pcaAnchor :: anchor
```

Comment:
The `anchor` type parameter seems redundant here. AFAIU you added that because sometimes you have `Anchor StandardCrypto` and sometimes `Anchor (EraCrypto (ShelleyLedger Era StandardCrypto))`, correct?

From the CLI point, it will be always `EraCrypto _ ~ StandardCrypto`.


Can you try to remove `anchor` parameter and try to use `StandardCrypto` everywhere? Are you expecting to support other `anchor` types here?

Reply:
It was more for the URL than the hash. For example, in the other PR the type was:
```haskell
  , proposalUrl :: !ProposalUrl
  , proposalHash :: !(L.SafeHash L.StandardCrypto L.AnchorData)
```
So I was opening the door for other packagings. But it may be that this doesn't happen inside a `Maybe`.
I will try to instantiate for now, and generalize if needed later, that is good practice